### DISCUSSION TITLE
Embed Subtab within New Dialog

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6275] [Help]
Hi,

I have created a subtable in a new Dialog Tab Pane , that subtable has to have the Id from the First Tab Pane. Quite unsure how to get it there. 

Attaching sample screenshot for clarity.

<img width="676" alt="Screenshot_3" src="https://user-images.githubusercontent.com/98172041/154289789-7461f5cc-c203-4c57-9980-ccabc0d7d66d.png">

**Comment by stixoffire** [comment]
In your order dialog.ts 
`mycustomer.form.field = myorder.form.property; `

Your customer tab should have
